quinn-proto/src/config.rs

```diff
@@ -518,12 +519,10 @@ where
     fn default() -> Self {
         let rng = &mut rand::thread_rng();
 
-        let mut token_key = vec![0; S::HmacKey::KEY_LEN];
-        rng.fill_bytes(&mut token_key);
-        Self::new(
-            S::HmacKey::new(&token_key)
-                .expect("HMAC key rejected random bytes; use ServerConfig::new instead"),
-        )
+        let mut master_key = vec![0u8; 64];
```

Comment:
I would like to know what would be a better way of doing this such that the size of the vector is not hard coded?

Reply:
We could use an associated constant on the trait, or modify the trait interface to take a fixed-size array rather than a slice. However, it's not clear to me that we actually need any particular length (see other comment) and in that case 64 bytes seems like a fine default.